### ISSUE TITLE
Navigation Affects Multiple GlideJS Instances

### ISSUE THREAD

**Issue by TJNevis** [jedrzejchalubek/Glide.js#65]
This is the first time I've reported an issue on Github.  I'll do my best.

I'm running 4 separate instances of GlideJS widgets, all instantiated with the same block of code, by their class name .slider.

//Sliders on About page
var glide = $('.slider').glide({
    arrowLeftText: '<span class="fa fa-angle-left fa-3x pull-left"></span>',
    arrowRightText: '<span class="fa fa-angle-right fa-3x pull-right"></span>',
    autoplay: false,
    afterTransition: function(){
        //For press section highlighting of logo
        if( this.parent[0].className.indexOf('press') != -1 ) {
            var slide = Math.abs( this.currentSlide );
            $('ul#press li').removeClass('active');
            $( $('ul#press').children()[slide] ).addClass('active');
        }
    }
}).data('api_guide');

The first slider scrolls through 8 quotes.  The fourth scrolls through 5 quotes and uses the code in the afterTransition callback.

The second and third sliders are the ones that are causing the issue I'm reporting.  The second slider holds one 'profile' slide for each of the people in the third slider and is visible, hiding the third slider - the third slider has 4 pictures of people and when clicked the second slider becomes visible, hiding the third slider.

In the case the third slider has 1 slide (with 4 people), clicking on a person makes the second slider visible - using the navigation on the second slider (paging through each of the 4 profiles), the third slider also moves - but since there is only one slide, it becomes blank.

$('#page-about .team .person').click(function() {
    $('section.team').addClass('visible').children('.profiles').addClass('visible');
    var glide = $('section.team .profiles .slider').glide().data('api_glide');
    glide.jump(this.dataset.person);
});

At first, I was thinking the arrows that were showing were the ones from the third slider, maybe a z-index problem, but the third slider never goes into line 229 in this.build(), which would attach the arrows (if configured).  Even if it was, the navigation is affecting 2 sliders at the same time.

I figured out a solution that works for this issue.  Changing line 445 from:
if (this.arrows.wrapper) {
to
if (this.slides.length > 1 && this.arrows.wrapper) {

I hope this hasn't been too confusing, I can provide more information if needed.  Typically I would just fix an issue and move on, but I really respect Glide.js it's really clean code, lightweight and I'd like to contribute if I can.

Another issue I notice is also with a 1 slide slider when adjusting screen widths (when testing a responsive website).  I may submit another ticket if I can pinpoint something down.

**Comment by racse1** [comment]
Hi everyone,
I was inspecting the Glide code and I found where bug come out. I wrote simple part of code that demonstrate it problem. I hope it'll help.
```js
function Slider(element, navigation) {
	this.element = element;

	if (navigation) {
		console.log('enable navigation');
		this.navigation();
	}

	if (this.navigation.enabled) {
		console.log(this.navigation.enabled);
	}
};

Slider.prototype.navigation = function() {
	this.navigation.enabled = true;
};

var a = new Slider($('slider-1'), true);
var b = new Slider($('slider-2'), false);
```
In all cases in console we'll see in log `true`, but navigation has been enabled only for first object. The problem in `Glide.prototype.arrows` method which set `Glide.prototype.arrows.wrapper` property.

Thanks.

**Comment by jedrzejchalubek** [comment]
Glide has been released in version v2.0.0. If the problem remains, please set up a new thread.
